script.js

```diff
@@ -0,0 +1,108 @@
+let selectedLi;
+let target = '';
+let ul = document.querySelector('.page__levels');
+let button = document.querySelector('.page__button');
+let startGame = document.querySelector('.page');
+let pageGame = document.querySelector('.game');
+let gameCards;
+let gameCardInners;
+let active = document.querySelector('.active');;
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/EmilLachinov8/JS-project/pullRequest?prid=7662088)
```suggestion
let active = document.querySelector('.active');
```